src/components/comparison/layers-panel/layers-control-panel.tsx

```diff
@@ -100,12 +144,12 @@ export const LayersControlPanel = ({
             }
             optionsContent={
               <LayerOptionsMenu
-                layerId={settingsLayerId}
-                showDeleteLayer={Boolean(layer?.custom)}
-                showLayerSettings={hasSettings && isLayerSelected}
-                onPointToLayerClick={() => {
+                layer={layer}
+                selected={isSelected}
+                showLayerSettings={hasSettings && isSelected}
```

Comment:
```suggestion
                hasSettings={hasSettings}
```
`isSelected` flag is already sent to the child component in the separate prop

Reply:
done